kategory-core/src/test/kotlin/kategory/data/SequenceKWTest.kt

```diff
@@ -31,6 +31,8 @@ class SequenceKWTest : UnitSpec() {
         }
 
         testLaws(
+            SemigroupLaws.laws(SequenceKW.semigroup(), { n -> SequenceKW(sequenceOf(n)) }, Eq.any()),
+            MonoidLaws.laws(SequenceKW.monoid(), { n -> SequenceKW(sequenceOf(n)) }, Eq.any()),
```

Comment:
Like `MapKW`, when `SequenceSemigroupIntance` combine two Sequences returns a `SequenceKW(sequence=kotlin.sequences.FlatteningSequence)` and a normal SequenceKW is like this `SequenceKW(sequence=kotlin.collections.ArraysKt...)`